### ISSUE TITLE
Accessibility test

### ISSUE THREAD

**Issue by wildfiremedia** [withastro/astro.build#1874]
Scanned Astro home page with Pa11y.

WCAG2AA.Principle4.Guideline4_1.4_1_2…

This emailinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.

#bd-email

Found one 1 issue, not plan to PR.
